Packs/Code42/Integrations/Code42/Code42.py

```diff
@@ -545,117 +560,6 @@ def get_file_category_value(key):
     return category_map.get(key, "UNCATEGORIZED")
 
 
-class ObservationToSecurityQueryMapper(object):
-    """Class to simplify the process of mapping observation data to query objects."""
-
-    # Exfiltration consts
-    _ENDPOINT_TYPE = "FedEndpointExfiltration"
-    _CLOUD_TYPE = "FedCloudSharePermissions"
-
```

Comment:
It is not. I was able to greatly simplify the query by just searching for the eventIds from the alert itself, rather than trying to reconstruct a query that matches.